### ISSUE TITLE
Inconsistancies between certian elements on whether they can be selected

### ISSUE THREAD

**Issue by RyanGar46** [modrinth/knossos#538]
### Environment

Website

### Describe the bug

Some elements like log out and change theme buttons can't be selected, but almost every other button can be.

![Screenshot_20220614_141550](https://user-images.githubusercontent.com/67124977/173661276-9f8aa84f-ff3a-4d0a-9e9c-d753756d31ef.png)

### Steps To Reproduce

1. Go to a user's profile
2. Select the entire page (CTRL + A)
3. You will see that some buttons aren't selected

### Expected behavior

The buttons should be consistently selectable. Either they should or they shouldn't be selectable.

**Comment by falseresync** [comment]
This happens because some buttons are <a> link tags and some are <button> button tags, because some are just navigation and some are functional

Ctrl + A selects all text components, which links obviously are, but true buttons are block components

It would be unnecessarily complicated to use either only links or only buttons, and in design hierarchy was deemed more important than semantics

**Comment by Prospector** [comment]
May be a browser thing or something has changed, but everything is selectable on Chrome, except for the checkboxes because they are easy to click and drag accidentally.
![image](https://user-images.githubusercontent.com/6166773/211478330-cb7709e0-082e-4990-a723-53bd2428bc49.png)
